--- PHP_CodeSniffer-2.0.0/usr/share/pear/PHP/CodeSniffer.php~	2014-12-18 15:30:31.000000000 +0200
+++ PHP_CodeSniffer-2.0.0/usr/share/pear/PHP/CodeSniffer.php	2014-12-18 15:31:17.858092309 +0200
@@ -1559,7 +1559,7 @@
                 $testPath = $path;
             }
 
-            $pattern = '`'.$pattern.'`i';
+            $pattern = '`'.$pattern.'`';
             if (preg_match($pattern, $testPath) === 1) {
                 return true;
             }
